src/styles/main.scss

```diff
@@ -1,3 +1,59 @@
+$days: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+$length--month: 31;
+$gap: 1px;
+$size: 100px;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  width: 7 * ($size + $gap);
+  flex-wrap: wrap;
+  gap: $gap;
+  margin: 10px;
+
+  &__day {
+    background-color: #eee;
+    height: $size;
+    width: $size;
+    box-sizing: border-box;
+    border: #000 solid 1px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    gap: $gap;
+
+    &:hover {
+      background-color: pink;
+      cursor: pointer;
+      transform: translateY(-20px);
+      transition-duration: 0.5s;
```

Comment:
don't add transition on hover, it will work oneway, also use shorthand syntax instead

https://github.com/mate-academy/layout_calendar/assets/106310796/934addfa-7959-46f1-9e1d-a53b68d15b0e

Reply:
9'th point in checklist